webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/score/ScoreService.scala

```diff
@@ -136,19 +138,21 @@ class ScoreServiceImpl(globalScoreRepository: GlobalScoreRepository, scoreReposi
     } yield {}
 
   }
-  def cleanScore(name: String): IOResult[Unit] = {
+  def cleanScore(name: String)(implicit qc: QueryContext): IOResult[Unit] = {
     for {
       _         <- cache.update(_.map { case (id, gscore) => (id, gscore.copy(details = gscore.details.filterNot(_.scoreId == name))) })
       newScores <- scoreCache.updateAndGet(_.map { case (id, scores) => (id, scores.filterNot(_.scoreId == name)) })
       _         <- update(newScores)
     } yield {}
   }
 
-  def update(newScores: Map[NodeId, List[Score]]): IOResult[Unit] = {
+  def update(newScores: Map[NodeId, List[Score]])(implicit qc : QueryContext): IOResult[Unit] = {
```

Comment:
same for the queryContext, I don't think we want that to dependant of a tenant view, else the cache will become inconsistant